tests/test_plugin.py

```diff
@@ -33,3 +33,12 @@ def test_destroy(fxa_client, fxa_account):
     result = testdir.runpytest()
     result.assert_outcomes(error=1, passed=1)
     assert 'ClientError: Unknown account' in result.stdout.str()
+
+
+def test_commandline_email_option(testdir):
+    testdir.makepyfile("""
+        import pytest
+        def test_pass(fxa_account): pass
```

Comment:
Can you add an assert to check that `fxa_account.email` is what you're expecting? Also, could you add a test for the environment variable?

Reply:
The email is available via `fxa_account.email`.

Reply:
Something like this?
``` 
def test_account(fxa_account):
    assert 'testemail@restmail.net' in fxa_account.email
```